Ziti.NET.Standard/msvc_build.bat

```diff
@@ -0,0 +1,102 @@
+@echo off
+
+ECHO Building the solution for the Ziti.NET.standard.dll
+
+SET Ziti_Net_HOME=%~dp0
+
+ECHO msbuild %Ziti_Net_HOME%..\Ziti.NuGet.sln /property:Configuration=Release /property:Platform=x64
+
+msbuild %Ziti_Net_HOME%..\Ziti.NuGet.sln /property:Configuration=Release /property:Platform=x64
+
+SET ACTUAL_ERR=%ERRORLEVEL%
+if %ACTUAL_ERR% NEQ 0 (
+    echo.
+    echo Build of %Ziti_Net_HOME%..\Ziti.NuGet.sln for Platform=x64 failed
+    echo.
+    goto FAIL
+) else (
+    echo.
+    echo result of msbuild for Platform=x64: %ACTUAL_ERR%
+)
+
+ECHO msbuild %Ziti_Net_HOME%..\Ziti.NuGet.sln /property:Configuration=Release /property:Platform=x86
+
+msbuild %Ziti_Net_HOME%..\Ziti.NuGet.sln /property:Configuration=Release /property:Platform=x86
+
+SET ACTUAL_ERR=%ERRORLEVEL%
+if %ACTUAL_ERR% NEQ 0 (
+    echo.
+    echo Build of %Ziti_Net_HOME%..\Ziti.NuGet.sln for Platform=x86 failed
+    echo.
+    goto FAIL
+) else (
+    echo.
+    echo result of msbuild for Platform=x86: %ACTUAL_ERR%
+)
+
+ECHO msbuild %Ziti_Net_HOME%..\Ziti.NuGet.sln /property:Configuration=Release
+
+msbuild %Ziti_Net_HOME%..\Ziti.NuGet.sln /property:Configuration=Release
+
+SET ACTUAL_ERR=%ERRORLEVEL%
+if %ACTUAL_ERR% NEQ 0 (
+    echo.
+    echo Build of %Ziti_Net_HOME%..\Ziti.NuGet.sln failed
+    echo.
+    goto FAIL
+) else (
+    echo.
+    echo result of msbuild: %ACTUAL_ERR%
+)
+
+SET NUGET_PATH=%Ziti_Net_HOME%..\NuGet
+
+mkdir %NUGET_PATH%
+
+ECHO dotnet pack %Ziti_Net_HOME%..\Ziti.NuGet.sln --configuration Release --output %Ziti_Net_HOME%
+
+dotnet pack %Ziti_Net_HOME%..\Ziti.NuGet.sln --configuration Release --output %Ziti_Net_HOME%
+
+SET ACTUAL_ERR=%ERRORLEVEL%
+if %ACTUAL_ERR% NEQ 0 (
+    echo.
+    echo nuget pack for %BUILD_VERSION% failed
+    echo.
+    goto FAIL
+) else (
+    echo.
+    echo result of dotnet pack: %ACTUAL_ERR%
+)
+
+FOR /F "delims= " %%i IN ('ls -rt Ziti.NET.Standard.*.nupkg ^| tail -n 1') DO set NUPKG_FILE=%%i
```

Comment:
can you add a comment that this expects 'tail' to be on your path to work? just a comment in the bat file is good enough for now